easybuild/easyconfigs/p/pyFFTW/pyFFTW-0.11.1-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonPackage'
+
+name = 'pyFFTW'
+version = '0.11.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/pyFFTW/pyFFTW'
+description = """A pythonic wrapper around FFTW, the FFT library, presenting a unified interface
+ for all the supported transforms."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+dependencies = [
+    ('Python', '3.6.6'),
+]
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['05ea28dede4c3aaaf5c66f56eb0f71849d0d50f5bc0f53ca0ffa69534af14926']
```

Comment:
Please move this part between `toolchain` and `dependencies`. It is not a strict error but we try to keep some common structure between the easyconfigs.